### PR TITLE
added healthcheck

### PR DESCRIPTION
**Pull request by temitope777**
### Trello card
https://trello.com/c/cgESA7Zy/862-implement-aks-healthcheck-school-experience
### Context
add healthcheck endpoint to app
### Changes proposed in this pull request
return 200 ok 
### Guidance to review
visit /check endpoiint

### CODE REVIEW

File: .github/common_environment.yml
Comment (saliceti) [inline .github/common_environment.yml:1]: This change is not related to this PR. Can you move it to another commit at least?

File: .github/common_environment.yml [hunk 1]
```diff
@@ -5,7 +5,7 @@
   SLACK_FOOTER:          School Experience
   DOCKER_REPOSITORY:     ghcr.io/dfe-digital/schools-experience
   DOMAIN:                london.cloudapps.digital
```
Comment (saliceti) [inline .github/common_environment.yml:7]: I believe this variable is not is use anymore

**Comment by saliceti** [comment]
Can we use the new healthcheck in terraform now?

File: .github/common_environment.yml
Comment (temitope777) [reply .github/common_environment.yml:1]: removed

File: .github/common_environment.yml (hunk 1 above)
Comment (temitope777) [reply .github/common_environment.yml]: will be removed in another PR

**Review by saliceti — APPROVED** [review]
Tested successfully: https://get-school-experience-review-pr-3054.test.teacherservices.cloud/check
